File: medico/Domain/Medico.Domain/Models/SelectableList.cs
using System;
using System.Collections.Generic;

namespace Medico.Domain.Models
{
    public class SelectableList : Entity
    {
        public bool IsPredefined { get; set; }

        public Guid? LibrarySelectableListId { get; set; }

        public SelectableList LibrarySelectableList { get; set; }

        public int? Version { get; set; }

        public Guid? CompanyId { get; set; }

        public Company Company { get; set; }

        public string JsonValues { get; set; }

        public SelectableListCategory Category { get; set; }

        public Guid CategoryId { get; set; }

        public string Title { get; set; }

        public bool IsActive { get; set; }

        public List<TemplateSelectableList> TemplateSelectableLists { get; set; }

        public List<SelectableList> LibraryRelatedSelectableLists { get; set; }
    }
}

File: medico/Services/Medico.Api/ClientApp/src/app/_models/selectableList.ts
import { SelectableListValue } from './selectableListValue';

export class SelectableList {
    id: string;
    companyId: string;
    title: string;
    selectableListValues: SelectableListValue[];
    isActive: boolean;
    categoryId: string;
    isPredefined: boolean;
    version: number | null;
    librarySelectableListId: string;

    constructor() {
        this.isActive = true;
        this.selectableListValues = [];  
    }
}